pkg/controller/registry/reconciler/grpc_test.go

```diff
@@ -374,11 +374,6 @@ func TestGetPodImageID(t *testing.T) {
 		pod         *corev1.Pod
 		result      string
 	}{
-		{
-			description: "no pod status: return nothing",
-			pod:         &corev1.Pod{Status: corev1.PodStatus{ContainerStatuses: []corev1.ContainerStatus{}}},
-			result:      "",
-		},
```

Comment:
This was removed since we moved the input validation from `GetPodImageID` correct?

Reply:
yes